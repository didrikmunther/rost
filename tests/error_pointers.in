fn main() {
	let a = 5
	let b = &a
	let c = &b
	let d = c + b
}