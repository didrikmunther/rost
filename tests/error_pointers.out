  --> ./tests/error_pointers.in:5:10, ./tests/error_pointers.in:5:12, ./tests/error_pointers.in:5:14 => CompilerError
4 |     let c = &b
5 |     let d = c + b
  |             ^ ^ ^
  |             │ │ └─ Other type is *int
  |             │ └─ Operator Plus is not defined for types.
  |             └─ Incompatible types in binary expression: **int
6 | }